components/ActionSheet/src/private/MDCActionSheetItemTableViewCell.m

```diff
@@ -67,19 +88,21 @@ - (void)commonMDCActionSheetItemViewInit {
   if (_itemAction.image || _addLeadingPadding) {
     leadingConstant = kTitleLeadingPadding;
   }
-  [_actionLabel.topAnchor constraintEqualToAnchor:self.contentView.topAnchor
+  [_actionLabel.topAnchor constraintEqualToAnchor:_contentContainerView.topAnchor
                                          constant:kActionItemTitleVerticalPadding]
       .active = YES;
-  [_actionLabel.bottomAnchor constraintEqualToAnchor:self.contentView.bottomAnchor
-                                            constant:-kActionItemTitleVerticalPadding]
-      .active = YES;
-  _titleLeadingConstraint = [_actionLabel.leadingAnchor
-      constraintEqualToAnchor:self.contentView.layoutMarginsGuide.leadingAnchor
-                     constant:leadingConstant];
+  NSLayoutConstraint *labelBottomConstraint =
+      [_actionLabel.bottomAnchor constraintEqualToAnchor:_contentContainerView.bottomAnchor
+                                                constant:-kActionItemTitleVerticalPadding];
+  labelBottomConstraint.priority = UILayoutPriorityDefaultHigh;
```

Comment:
Is this needed? I see this as the only one with priority?

Reply:
Lowering the priority is required to not break constraints when setting content edge insets. If this is kept at `.required`, then the distance from the label to bottom is incompatible.